gem/lib/rbi-central/static/context.rb

```diff
@@ -19,8 +19,12 @@ def initialize(gem, annotations_file, color:, bundle_config: {})
       def init!
         super
 
-        add_gem_dependency("sorbet", version: ">= 0.5.10109")
-        add_gem_dependency("tapioca", version: ">= 0.9.2")
+        add_gem_dependency("sorbet",  version: "= #{::Gem.loaded_specs["sorbet"].version}")
+        add_gem_dependency("tapioca", version: "= #{::Gem.loaded_specs["tapioca"].version}")
+
+        # To be deleted once shopify-money has a direct dependency on bigdecimal
+        # https://github.com/Shopify/money/pull/415
+        add_gem_dependency("bigdecimal", version: ">= 3.0")
```

Comment:
We'll need this until https://github.com/Shopify/money/pull/415 is shipped and released.